Tests/TestingApps/PaywallsTester/PaywallsTester/SimpleApp.swift

```diff
@@ -2,22 +2,68 @@
 //  SimpleApp.swift
 //  SimpleApp
 //
-//  Created by Nacho Soto on 5/30/23.
+//  Created by James Borthwick on 4/25/24.
 //
 
 import SwiftUI
 
 @main
 struct SimpleApp: App {
```

Comment:
this name is suuuper outdated, we set up the name "SimpleApp" when we were thinking about having a simple app that didn't have anything but a paywall on it for testing. Might as well rename it to something more appropriate

Reply:
I used `PaywallsPreview`, which I think is a better description than `PaywallsTester`.